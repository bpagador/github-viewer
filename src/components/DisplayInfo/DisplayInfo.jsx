import React from 'react';
import PropTypes from 'prop-types';

const DisplayInfo = ({ profile }) => {
  
  return (
    <section>

      <header>
        {profile.name}
      </header>

      <div>
      
        <h4>Followers: {profile.followers}</h4>
  
        <h5>Following: {profile.following}</h5>
    
      </div>

      <p>
        <a href={profile.html_url}>{profile.login}</a>
      </p>
      
    </section>
  );
};

DisplayInfo.propTypes = {
  profile: PropTypes.object.isRequired
};

export default DisplayInfo;
