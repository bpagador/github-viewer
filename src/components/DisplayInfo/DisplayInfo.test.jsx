import React from 'react';
import { shallow } from 'enzyme';
import DisplayInfo from './DisplayInfo';

describe('DisplayInfo', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <DisplayInfo 
        profile={{}}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
