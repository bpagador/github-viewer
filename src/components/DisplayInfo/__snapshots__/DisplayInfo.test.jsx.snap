// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayInfo matches a snapshot 1`] = `
<section>
  <header />
  <div>
    <h4>
      Followers: 
    </h4>
    <h5>
      Following: 
    </h5>
  </div>
  <p>
    <a />
  </p>
</section>
`;
